apps/src/applab/commands.js

```diff
@@ -539,10 +539,15 @@ applabCommands.setStrokeWidth = function (opts) {
 
 // Returns an rbg or rgba color string that can be used as a parameter to other functions.
 applabCommands.rgb = function (opts) {
-  apiValidateTypeAndRange(opts, 'rgb', 'number', opts.r, 'number', 0, 255);
-  apiValidateTypeAndRange(opts, 'rgb', 'number', opts.g, 'number', 0, 255);
-  apiValidateTypeAndRange(opts, 'rgb', 'number', opts.b, 'number', 0, 255);
+  apiValidateTypeAndRange(opts, 'rgb', 'number', opts.r, 'number');
+  apiValidateTypeAndRange(opts, 'rgb', 'number', opts.g, 'number');
+  apiValidateTypeAndRange(opts, 'rgb', 'number', opts.b, 'number');
   apiValidateTypeAndRange(opts, 'rgb', 'number', opts.a, 'number', 0, 1, OPTIONAL);
+
+  // Convert any decimal values into integers between 0 and 255
+  opts.r = Math.min(255, Math.max(0, parseInt(opts.r)));
+  opts.g = Math.min(255, Math.max(0, parseInt(opts.g)));
+  opts.b = Math.min(255, Math.max(0, parseInt(opts.b)));
```

Comment:
Might want to specify a radix for parseInt (i.e. `parseInt(opts.r, 10)`). I'm a little surprised our linter doesn't look for this. See http://eslint.org/docs/rules/radix for why (or run `parseInt(017)` and see if the result is what you expect.